kiali.go

```diff
@@ -1,3 +1,18 @@
+// Kiali
+//
+// Kiali project, observability for the Istio service mesh
+//
+//     Schemes: http
```

Comment:
Minor: "Or https, as by default OS traffic is secured."

Reply:
Actually the API not works with https only http